src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax.service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    private static final String OUTPUT_FILE_PATH
+            = "src/main/java/core/basesyntax/resources/outputFile.csv";
+    private static String date;
+    private WriterServiceImpl writerService;
+
+    @BeforeEach
+    void setUp() {
+        writerService = new WriterServiceImpl();
```

Comment:
Please do not create new instance of service for each test unless it's really necessary.
On bigger project your test suit may run for several minutes and you won't wish it to run even longer due to creating of a bunch of new objects :)

Reply:
Still in BeforeEach section, you have it initialized on line 16, that's enough